src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorEngine.java

```diff
@@ -327,6 +333,22 @@ private VariantContext getCorrectVariantContextForReference(final VariantContext
         }
     }
 
+    /**
+     * Create a new {@link VariantContext} which will match the given Reference if there is a mismatch for input between the B37 reference and the HG19 reference.
+     * @param variant A {@link VariantContext} object containing the variant to convert.
+     * @return A {@link VariantContext} whose contig has been transformed to HG19 if requested by the user.  Otherwise, an identical variant.
+     */
+    VariantContext getCorrectVariantContextForOutput(final VariantContext variant) {
+        if ( mustRevertVariantContigsFromHg19ToB37 ) {
+            final VariantContextBuilder vcb = new VariantContextBuilder(variant);
+            vcb.chr(FuncotatorUtils.convertHG19ContigToB37Contig(variant.getContig()));
+            return vcb.make();
```

Comment:
According to Codecov, this new reversion code is not covered by tests. Can you add a test with a b37 vcf and hg19 reference to exercise this case?

Reply:
As i have discussed elsewhere, you can see the 5 integration test files that have changed to now be valid as a result (which rests on this code) I think codecov is wrong in this case...